src/style.css

```diff
@@ -0,0 +1,98 @@
+@font-face {
+  src: url(fonts/Avenir-Book.ttf);
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 400;
+}
+
+* {
+  box-sizing: border-box;
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.search-string {
+  background: #fff;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  position: relative;
+  width: 100%;
+  margin-top: 20px;
+}
+
+.search-string:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search-string:focus {
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.image-big {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.input-line {
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  outline: none;
+  width: 100%;
+  color: #3d4e61;
+  line-height: 21.86px;
+  font-weight: 400;
+}
+
+.input-line-big {
+  padding: 24px 0 24px 62px;
+  line-height: 21.86px;
+  font-family: inherit;
+  height: 70px;
+}
+
+.input-line-big::placeholder {
+  font-size: 16px;
+  line-height: 21.86px;
+  font-weight: 300;
+  font-family: inherit;
+  color: #3d4e61;
+}
+
+.image-little {
+  position: absolute;
+  width: 11px;
+  height: 11px;
+  left: 13px;
+  top: 15px;
+}
+
+.input-line-little {
+  padding: 11px 0 11px 33px;
+  font-size: 14px;
+  line-height: 19px;
+  font-weight: 400;
+  font-family: inherit;
```

Comment:
move `font-weight` and `font-family` to the input-line you don't need duplicate it

Reply:
done